python/ExodusNNTrainer/NeuralNet_to_XML.py

```diff
@@ -0,0 +1,71 @@
+import torch
```

Comment:
neads a header with hash-bang and comments

Reply:
I would like these files to be in a python directory: python/stochastic_tools

These files also need unit testing of their own